jormungandr-lib/src/interfaces/vote/mod.rs

```diff
@@ -247,12 +368,36 @@ impl From<vote::TallyResult> for TallyResult {
     }
 }
 
+impl From<chain_vote::TallyResult> for TallyResult {
+    fn from(this: chain_vote::TallyResult) -> Self {
+        // TODO: is it safe to unwrap here?
+        Self {
+            results: this.votes.iter().map(|w| w.unwrap().into()).collect(),
+            options: (this.options.start as u8..this.options.end as u8),
+        }
+    }
+}
+
```

Comment:
@mzabaluev what do you think should be done here (both for results and options )?

Reply:
I think it's safer to fall back the `None` votes into 0, and use the `TryFrom` impl to narrow the bit width:

```rust
        Self {
            results: this.votes.iter().map(|w| w.unwrap_or(0).into()).collect(),
            options: (this.options.start.try_into().unwrap()..this.options.end.try_into().unwrap()),
        }
```